src/status_im/ui/screens/ens/views.cljs

```diff
@@ -380,80 +363,195 @@
+
+(defn- term-point [content]
+  [react/view {:style {:flex 1 :margin-top 24 :margin-horizontal 16 :flex-direction :row}}
+   [react/view {:style {:width 16 :margin-top 8}}
+    [react/view {:style {:background-color colors/gray :width 4 :height 4 :border-radius 25}}]]
+   [react/text {:style {:flex 1 :font-size 15}}
+    content]])
+
+(defn- etherscan-url [address]
+  (str "https://etherscan.io/address/" address))
+
+(views/defview terms []
+  (views/letsubs [{:keys [contract]} [:get-screen-params :ens-terms]]
+    [react/scroll-view {:style {:flex 1}}
+     [status-bar/status-bar {:type :main}]
+     [toolbar/simple-toolbar
+      (i18n/label :t/ens-terms-registration)]
+     [react/view {:style {:height 136 :background-color colors/gray-lighter :justify-content :center :align-items :center}}
+      [react/text {:style {:text-align :center :typography :header :letter-spacing -0.275}}
+       (i18n/label :t/ens-terms-header)]]
      [react/view
-      [button {:on-press #(re-frame/dispatch [:ens/set-state username :registering])}
-       (i18n/label :t/retry)]
-      [button {:background? false
-               :on-press    #(re-frame/dispatch [:ens/clear-cache-and-navigate-back])}
-       (i18n/label :t/cancel)]]
-     [button {:on-press #(re-frame/dispatch [:ens/clear-cache-and-navigate-back])}
-      (i18n/label :t/ens-got-it)])])
-
-(views/defview registration-pending [{:keys [state custom-domain?] :as props} usernames]
-  (views/letsubs [usernames [:multiaccount/usernames]]
-    [react/view {:style {:flex 1}}
-     [react/scroll-view {:style {:flex 1}}
-      [react/view {:style {:flex 1}}
-       [react/view {:style {:flex 1 :align-items :center :justify-content :center}}
-        [icon props]
-        [username-input props usernames]
-        [react/view {:style {:height 36 :align-items :center :justify-content :space-between :padding-horizontal 12
-                             :margin-top 24 :margin-horizontal 16 :border-color colors/gray-lighter :border-radius 20
-                             :border-width 1 :flex-direction :row}}
-         [react/text {:style {:font-size 12 :typography :main-medium}}
-          (domain-label custom-domain?)]
-         [react/view {:flex 1 :min-width 24}]
-         (when-not (= state :registering)
-           ;; Domain type is not shown during registration
-           [react/touchable-highlight {:on-press #(re-frame/dispatch [:ens/switch-domain-type])}
-            [react/text {:style {:color colors/blue :font-size 12 :typography :main-medium} :number-of-lines 2}
-             (domain-switch-label custom-domain?)]])]]
-       (if (= state :registering)
-         [registration props]
-         [react/text {:style {:flex 1 :margin-top 16 :margin-horizontal 16 :font-size 14 :text-align :center}}
-          (help-message state custom-domain?)])]]
-     (when (= state :registering)
-       [registration-bottom-bar props])]))
+      [term-point
+       (i18n/label :t/ens-terms-point-1)]
```

Comment:
This is repetitive but necessary because we want all i18n labels explicitly names and not constructed, right?

Reply:
yes and also I am going to change them all in a future PR because the copy has been revisited, and these labels are not good for translation because if you change the meaning of the translation it is not reflected